charts/argo-cd/templates/redis/service.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   name: {{ template "argo-cd.redis.fullname" . }}
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.redis.name "name" .Values.redis.name) | nindent 4 }}
+  {{- with .Values.redis.serviceAnnotations }}
```

Comment:
Please use `.Values.redis.service.annotations` as other components implement it this way.